build/devbase/vendor.sh

```diff
@@ -44,16 +44,7 @@ function download_verify()
 }
 
 # Parallelize the builds below based on the number of cpus.
-ncpus=1
-case $(uname) in
-    Linux)
-        ncpus=$(grep processor /proc/cpuinfo | wc -l)
-	;;
-    Darwin)
-	ncpus=$(sysctl -q -n hw.logicalcpu)
-	;;
-esac
-
+ncpus=$(go run $(pwd -P)/build/devbase/ncpu.go)
```

Comment:
Do we need `$(pwd -P)` in this context (as opposed to simply `$PWD` or `.`)? I thought `go run` was fine with relative paths.